packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1212,7 +1286,7 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @protected
   @mustCallSuper
   void handleThumbPress() {
-    _checkForValidScrollPosition();
+    _debugCheckHasValidScrollPosition();
```

Comment:
Methods called `_debug*` should usually be wrapped in an assert and return a bool.